contracts/core/PriceManager.sol

```diff
@@ -31,6 +31,12 @@ contract PriceManager is IPriceManager, Ownable, Constants {
         isInitialized[_token] = true;
     }
 
+    function setMaxLeverage(address _token, uint256 _maxLeverage) external onlyOwner {
+        require(isInitialized[_token] == true, "can only modify maxLeverage for existing tokens");
```

Comment:
why do we need to check isInitialized? initialize work at first

Reply:
just to avoid wrong param passed, admin may copy paste a wrong addr